### PR TITLE
{bio,tools}[iccifort/2020.1.217,intel/2020a] CheckM v1.1.3, Pysam v0.16.0.1 w/ Python 2.7.18 + 3.8.2

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/Pysam/Pysam-0.16.0.1-iccifort-2020.1.217.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+multi_deps = {'Python': ['3.8.2', '2.7.18']}
+
+dependencies = [
+    ('ncurses', '6.2'),
+    ('cURL', '7.69.1'),
+    ('XZ', '5.2.5'),
+]
+
+# required because we're building Python packages using Intel compilers on top of Python built with GCC
+check_ldshared = True
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pysam/Pysam-0.16.0.1-iccifort-2020.1.217.eb:37]: @Darkless012 It should be OK to remove this (the `PythonPackage` easyblock is now smart enough to auto-enable this when needed, see https://github.com/easybuilders/easybuild-easyblocks/pull/1788)

File: easybuild/easyconfigs/p/Pysam/Pysam-0.16.0.1-iccifort-2020.1.217.eb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pysam/Pysam-0.16.0.1-iccifort-2020.1.217.eb:45]: this is the default for `PythonPackage`, so unless there's something more specific to check for (scripts in `bin` for example), there's no need to specify `sanity_check_paths` like this.

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/Pysam/Pysam-0.16.0.1-iccifort-2020.1.217.eb (hunk 2 above)
Comment (Darkless012) [reply easybuild/easyconfigs/p/Pysam/Pysam-0.16.0.1-iccifort-2020.1.217.eb:45]: oh, right :) copy-pasted from older version. Thanks for noticing

File: easybuild/easyconfigs/c/CheckM/CheckM-1.1.3-intel-2020a-Python-3.8.2.eb [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+sanity_check_paths = {
+    'files': ['bin/checkm'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = ["checkm test %(builddir)s/checkm_test_results"]
+
+sanity_pip_check = True
+
+moduleclass = 'tools'
```
Comment (verdurin) [inline easybuild/easyconfigs/c/CheckM/CheckM-1.1.3-intel-2020a-Python-3.8.2.eb:65]: @Darkless012 oughtn't this to be `bio`?

**Review by verdurin — DISMISSED** [review]
Please also address all the comments by @boegel

File: easybuild/easyconfigs/c/CheckM/CheckM-1.1.3-intel-2020a-Python-3.8.2.eb (hunk 3 above)
Comment (Darkless012) [reply easybuild/easyconfigs/c/CheckM/CheckM-1.1.3-intel-2020a-Python-3.8.2.eb:65]: All past easyconfigs are `tools`. Makes sence to move it to bio though?

Comment (Darkless012) [reply easybuild/easyconfigs/c/CheckM/CheckM-1.1.3-intel-2020a-Python-3.8.2.eb:65]: Maybe merge and then update all .eb to same `moduleclass`?

Comment (verdurin) [reply easybuild/easyconfigs/c/CheckM/CheckM-1.1.3-intel-2020a-Python-3.8.2.eb:65]: @Darkless012 It looks pretty unambiguously like `bio` to me. Personally I would just change it for this update and leave the old ones as they are. @boegel ?

Comment (boegel) [reply easybuild/easyconfigs/c/CheckM/CheckM-1.1.3-intel-2020a-Python-3.8.2.eb:65]: Change it to `bio` here, and open a *separate* PR to fix the `moduleclass` in the existing `CheckM` easyconfigs to `bio`.

Comment (Darkless012) [reply easybuild/easyconfigs/c/CheckM/CheckM-1.1.3-intel-2020a-Python-3.8.2.eb:65]: ack

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/124422e0ab385ee0eed03cd67dadf0c7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3570.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/a0f0c7063cc62dddcfe660f8b23ffcc9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3401.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/9d48fff4f7dbe556f6f4d2af6c550320 for a full test report.

File: easybuild/easyconfigs/c/CheckM/CheckM-1.1.3-intel-2020a-Python-3.8.2.eb (hunk 3 above)
Comment (boegel) [reply easybuild/easyconfigs/c/CheckM/CheckM-1.1.3-intel-2020a-Python-3.8.2.eb]: moduleclass fixed in #11662

**Comment by boegel** [comment]
Going in, thanks @Darkless012!

**Pull request merged**